Boolder/Models/Circuit.swift

```diff
@@ -233,11 +233,17 @@ extension Circuit {
     }
     
     var firstProblem: Problem? {
-        let query = Table("problems")
+        let queryD = Table("problems")
+            .filter(Problem.circuitId == self.id)
+            .filter(Problem.circuitNumber == "D")
```

Comment:
let's use a constant for the letter D, so that it's easier to refactor in the future